src/index.js

```diff
@@ -1,3 +1,5 @@
+import { parseGanglion } from 'openbci-utilities/dist/utilities'
+import constants from 'openbci-utilities/dist/constants'
```

Comment:
Here you can just do:
`import k from 'openbci-utilities/dist/constants';`
And remove lines 15 and 16.
Also, don't forget those semicolons :)